presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/README.md

```diff
@@ -0,0 +1,148 @@
+# Presto planner and optimizer tests
+
+Testing planner and optimizer functionality generally requires generating a
+plan and then validating it against a set of expectations. The challenge is
+expressing the expectations in a way that's concise and resilient to changes in
+the objects Presto uses to represent the plan and the code that generates the
+plan.
+
+Unfortunately, that list includes the following:
+
+* The parser
+* The analyzer
+* The planner
+* Some or all of the optimizers
+* The plan objects themselves
+
+The plan testing framework attempts to solve the problem of specifiying
+expectations about query plans in tests.
+
+## Quickstart
+
+If you want to see examples of basic use of the plan testing framework, look at
+TestPlanFramework.java. If you'd like to try to begin writing new tests, look
+at PlanMatchPattern.java to see what sorts of patterns are already
+well-supported by the framework.
+
```

Comment:
Does TestPlanFramework.java contain several examples? 
If yes, suggest:
TestPlanFramework.java contains examples of basic use of the plan testing framework. 
If no, suggest:
TestPlanFramework.java is an example of a basic plan testing framework.
Next sentence, suggest:
If you'd like to write new tests, see PlanMatchPattern.javs to see patterns that are well-supported by the framework.